apps/taxes/src/modules/taxjar/taxjar-webhook.service.ts

```diff
@@ -0,0 +1,45 @@
+import pino from "pino";
+import { OrderCreatedSubscriptionFragment, TaxBaseFragment } from "../../../generated/graphql";
+import { createLogger } from "../../lib/logger";
+import { ChannelConfig } from "../channels-configuration/channels-config";
+import { ProviderWebhookService } from "../taxes/tax-provider-webhook";
+import { TaxJarClient } from "./taxjar-client";
+import { TaxJarConfig } from "./taxjar-config";
+import { taxJarCalculateTaxes } from "./maps/taxjar-calculate-taxes-map";
+import { taxJarOrderCreated } from "./maps/taxjar-order-created-map";
+
+export class TaxJarWebhookService implements ProviderWebhookService {
+  client: TaxJarClient;
+  private logger: pino.Logger;
+
+  constructor(config: TaxJarConfig) {
+    const avataxClient = new TaxJarClient(config);
+
+    this.client = avataxClient;
+    this.logger = createLogger({
+      service: "TaxJarProvider",
+    });
+  }
+
+  async calculateTaxes(payload: TaxBaseFragment, channel: ChannelConfig) {
+    this.logger.debug({ payload, channel }, "calculateTaxes called with:");
+    const args = taxJarCalculateTaxes.mapPayload(payload, channel);
+    const fetchedTaxes = await this.client.fetchTaxForOrder(args);
+    this.logger.debug({ fetchedTaxes }, "fetchTaxForOrder response");
```

Comment:
same as before, logger bloats a little and make it harder to debug, I'd suggest to add blank lines between logger and actual logic

Reply:
you resolved my comment but I dont see the change

Reply:
Added